regression/testing_tool.py

```diff
@@ -61,9 +62,6 @@ def mark_test_as_knownbug(self, issue: str):
             to it"""
         raise NotImplementedError
 
-    def append_test_arg(self, arg: str):
```

Comment:
Why are you removing this method?

Reply:
because the additional test args could be passed via `ESBMC_OPTS` (https://github.com/esbmc/esbmc/pull/643#issuecomment-1069014994)